.travis.yml

```diff
@@ -7,12 +7,15 @@ matrix:
   include:
     - name: "Python 2.6"
       python: 2.6
+      dist: trusty        # required for legacy Python
```

Comment:
Wow, that really is reaching back into the past!  :)

Reply:
https://pythonclock.org/ :-)